MdeModulePkg/Core/Dxe/Dispatcher/Dispatcher.c

```diff
@@ -414,6 +414,7 @@ CoreDispatcher (
     //
     // If the dispatcher is running don't let it be restarted.
     //
+    PERF_FUNCTION_END (); // MS_CHANGE

```

Comment:
Please try to replace these with `MU_CHANGE` if you can before publishing the PR.